pytest-embedded-qemu/pytest_embedded_qemu/qemu_device.py

```diff
@@ -77,11 +93,24 @@ def qemu_default_args(self):
 
         return self.QEMU_DEFAULT_ARGS
 
+    def qmp_execute_cmd(self, execute, arguments=None):
+        response = None
+
+        async def h_r():
+            nonlocal response
+            qmp = QMPClient()
```

Comment:
could we reuse the QMPClient here?

Reply:
I put the `QMPClient` in the `__init__` function, but the `connect` and `disconnect` functions are still left in their current place. Since this code is asynchronous, there might be some potential conflicts.